ingestion_tools/dataset_configs/template.yaml

```diff
@@ -1,137 +1,188 @@
+      name: REQUIRED, STRING
+      primary_author_status: OPTIONAL, BOOLEAN (DEFAULT FALSE)
+    dates: REQUIRED
+      deposition_date: REQUIRED, STRING, DATE FORMAT YYYY-MM-DD
+      last_modified_date: REQUIRED, STRING, DATE FORMAT YYYY-MM-DD
+      release_date: REQUIRED, STRING, DATE FORMAT YYYY-MM-DD
+    ground_truth_status: REQUIRED, BOOLEAN (DEFAULT FALSE)
+    is_curator_recommended: OPTIONAL, BOOLEAN (DEFAULT FALSE)
+    method_type: REQUIRED, ENUM (manual, hybrid, automated)
+    version: OPTIONAL, STRING
+  sources: REQUIRED
+  - binning: OPTIONAL, INTEGER (DEFAULT 1) (Point, OrientedPoint, InstanceSegmentation)
+    columns: OPTIONAL, STRING (DEFAULT xyz) (Point)
+    delimiter: OPTIONAL, STRING (DEFAULT ',') (Point)
+    file_format: REQUIRED, STRING
+    filter_value: OPTIONAL, STRING (OrientedPoint, InstanceSegmentation)
+    glob_string: REQUIRED, STRING (REGEX)
+    is_visualization_default: OPTIONAL, BOOLEAN (DEFUALT false)
+    mask_label: OPTIONAL, INTEGER (DEFAULT 1) (SemanticSegmentationMask)
+    order: OPTIONAL, STRING (OrientedPoint, InstanceSegmentation)
+    shape: OPTIONAL, ENUM (InstanceSegmentation, OrientedPoint, Point, SegmentationMask, SemanticSegmentationMask)
+dataset_keyphotos: OPTIONAL
+- sources: REQUIRED
+  - literal: REQUIRED
+      value: REQUIRED
+        snapshot: REQUIRED, STRING
+        thumbnail: REQUIRED, STRING
+datasets: REQUIRED
+- metadata: REQUIRED
+    authors: SEE annotations.authors
+    cell_component: OPTIONAL
+      id: OPTIONAL, STRING (GOID)
+      name: REQUIRED, STRING
+    cell_strain: SEE cell_component
+    cell_type: SEE cell_component
+    cross_references: OPTIONAL
+      dataset_publications: OPTIONAL, STRING (DOIs)
+      related_database_entries: OPTIONAL, STRING (EMPIAR / EMDB IDs)
+    dataset_description: REQUIRED, STRING
+    dataset_identifier: REQUIRED, INTEGER
+    dataset_title: REQUIRED, STRING
+    dates: SEE annotations.dates
+    funding: OPTIONAL BUT RECOMMENDED
+    - funding_agency_name: OPTIONAL BUT RECOMMENDED, STRING
+      grant_id: OPTIONAL BUT RECOMMENDED, STRING
+    grid_preparation: OPTIONAL BUT RECOMMENDED, STRING
+    organism: OPTIONAL
+      name: REQUIRED, STRING
+      taxonomy_id: OPTIONAL, INTEGER
+    sample_preparation: OPTIONAL BUT RECOMMENDED, STRING
+    sample_type: REQUIRED, STRING
+    tissue: SEE cell_component
+  sources: REQUIRED (source_glob, source_multi_glob, destination_glob, literal)
+  - source_glob: OPTIONAL # only one finder type per list item, each can have a parent_filters
+      list_glob: REQUIRED, STRING (REGEX)
+      match_regex: OPTIONAL, STRING (REGEX) (DEFAULT '.*')
+      name_regex: OPTIONAL, STRING (REGEX) (DEFAULT '(.*)')
+    parent_filters: OPTIONAL
+      exclude: OPTIONAL
+        run: # can only exclude / include parent types (see IMPORTER_DEP_TREE)
+          - OPTIONAL, STRING (REGEX)
+      include: OPTIONAL
+        run: REQUIRED
+          - OPTIONAL, STRING (REGEX)
```

Comment:
dataset, dataset_keyphotos and runs will be the only components, that can't have run as its parent attribute.

Reply:
Noted! I've moved parent_filters to runs to make more sense and added some comments / additional `SEE ...` references to hopefully make the code make more sense.